### PR TITLE
chore(poker): use generic components for GitHub and GitLab

### LINKED ISSUE #6435: Poker scope phase: use generic components for GitHub and GitLab
Use newly created generic components from here #6406 for GitHub and GitLab integrations. See Jira and JiraServer for example.

### Acceptance Criteria (optional)
- GitHub and GitLab integrations using reusable components on scope phase of Poker meeting

**Estimated effort:** 2h

### PR DESCRIPTION
**Pull request by igorlesnenko**
Fixes #6435

How to test:

- Create poker meeting
- On the scope phase play with all integrations controls to make sure nothing is broken

### CODE REVIEW

**Comment by JimmyLv** [comment]
Hey @igorlesnenko , I'll be out next week, could you assign @Dschoordsch to review this PR? 😁

File: packages/client/components/GitLabScopingSearchBar.tsx [hunk 1]
```diff
@@ -1,63 +1,89 @@
...
+    <ScopingSearchBar currentFilters={currentFilters}>
+      <ScopingSearchHistoryToggle />
+      <ScopingSearchInput
+        placeholder={'Search GitLab issues...'}
+        queryString={queryString}
+        meetingId={meetingId}
+        linkedRecordName={'gitlabSearchQuery'}
+        onChange={() => {
+          if (!queryString) {
+            trackEvent('Started Poker Scope Search')
```
Comment (Dschoordsch) [inline packages/client/components/GitLabScopingSearchBar.tsx:78]: +1 could we always track this event? Then we could get rid of these callbacks.

**Review by Dschoordsch — CHANGES_REQUESTED** [review]

File: packages/client/components/GitLabScopingSearchBar.tsx (hunk 1 above)
Comment (igorlesnenko) [reply packages/client/components/GitLabScopingSearchBar.tsx:78]: @Dschoordsch this was only added for GitLab based on this discussion here: #6296
```
SendClientSegmentEventMutation(atmosphere, eventTitle, {
      viewerId,
      meetingId,
      service: 'gitlab'
    })
   ```
   
I can track this for all integrations. But I'm not sure if we really need these events for other integrations. Should we?
 
cc @nickoferrall

Comment (igorlesnenko) [reply packages/client/components/GitLabScopingSearchBar.tsx]: @Dschoordsch fixed

File: packages/client/components/ScopingSearchInput.tsx [hunk 2]
```diff
@@ -36,11 +38,14 @@ interface Props {
   queryString: string
   meetingId: string
   linkedRecordName: string
+  service: IntegrationProviderServiceEnum | 'parabol'
```
Comment (Dschoordsch) [inline packages/client/components/ScopingSearchInput.tsx:41]: -1 the underlying mutation does not support `parabol` as value
```
13:08:31 1|GraphQL Executor |   error: '{"errors":[{"message":"Variable \\"$service\\" got invalid value \\"parabol\\"; Expected type TaskServiceEnum.","locations":[{"line":3,"column":3}]}]}'
```

**Review by Dschoordsch — CHANGES_REQUESTED** [review]

File: packages/client/components/ScopingSearchInput.tsx [hunk 3]
```diff
@@ -36,11 +38,14 @@ interface Props {
   queryString: string
   meetingId: string
   linkedRecordName: string
+  service: IntegrationProviderServiceEnum | 'PARABOL'
```
Comment (Dschoordsch) [inline packages/client/components/ScopingSearchInput.tsx:41]: -1 We should use `TaskServiceEnum` from `__generated__/SendClientSegmentEventMutation.graphql` here instead.

**Review by Dschoordsch — CHANGES_REQUESTED** [review]

File: packages/client/components/ScopingSearchInput.tsx (hunk 3 above)
Comment (igorlesnenko) [reply packages/client/components/ScopingSearchInput.tsx]: ah totally make sense! Fixed

**Review by Dschoordsch — APPROVED** [review]

**Pull request merged**